src/index.css

```diff
@@ -0,0 +1,14 @@
+
+@import "./styles/card-style.css";
+@import "./styles/stars-style.css";
+
+* {
+  margin: 0;
```

Comment:
Don't reset margins using * selector, use specific tags.
Box-sizing is correct here.